From 0eeb4558d541fac40b486a224eb6c601b64115a1 Mon Sep 17 00:00:00 2001
From: Himal Ghimiray <himalg@codeaurora.org>
Date: Mon, 16 Oct 2017 20:38:51 +0530
Subject: set_bb_env: Start compiling user build .

-Adding User build compilation for apq8053, apq8053-32 and
apq8053-32-minimal.

Change-Id: I7d0ea5e406d0d19dc7298a0c9729fa44b14e3de3
---
 set_bb_env.sh | 29 +++++++++++++++++++++++++++++
 1 file changed, 29 insertions(+)

diff --git a/set_bb_env.sh b/set_bb_env.sh
index 9172da0..c718ce4 100755
--- a/set_bb_env.sh
+++ b/set_bb_env.sh
@@ -325,9 +325,19 @@ function build-8053-perf-image() {
   cdbitbake machine-image
 }
 
+function build-8053-user-image() {
+  unset_bb_env
+  export MACHINE=apq8053
+  export DISTRO=msm-user
+  export VARIANT=user
+  export PRODUCT=base
+  cdbitbake machine-image
+}
+
 build-all-8053-images() {
   build-8053-image
   build-8053-perf-image
+  build-8053-user-image
 }
 
 # 8053-32 commands
@@ -346,6 +356,14 @@ function build-8053-32-perf-image() {
   cdbitbake machine-image
 }
 
+function build-8053-32-user-image() {
+  unset_bb_env
+  export MACHINE=apq8053-32
+  export DISTRO=msm-user
+  export VARIANT=user
+  cdbitbake machine-image
+}
+
 function build-8053-32-minimal-image() {
   unset_bb_env
   export MACHINE=apq8053-32
@@ -362,11 +380,22 @@ function build-8053-32-perf-minimal-image() {
   cdbitbake machine-minimal-image
 }
 
+function build-8053-32-user-minimal-image() {
+  unset_bb_env
+  export MACHINE=apq8053-32
+  export DISTRO=msm-user
+  export VARIANT=user-minimal
+  cdbitbake machine-minimal-image
+}
+
+
 build-all-8053-32-images() {
   build-8053-32-minimal-image
   build-8053-32-image
   build-8053-32-perf-minimal-image
   build-8053-32-perf-image
+  build-8053-32-user-minimal-image
+  build-8053-32-user-image
 }
 
 # 8096 commands
-- 
cgit v1.1

